src/SearchResults.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import SearchResult from "./SearchResult";
+import TableHeading from "./TableHeading";
+
+const SearchResults = props => {
+  return (
+    <div>
+      <table className="table table-striped table-bordered table-hover text-center">
+        <TableHeading />
+        <tbody>
+          {props.results.map((info, index) => (
+            <SearchResult info={info} index={index} />
```

Comment:
It's really fantastic that you thought to extract `SearchResult` to a separate component!